cookbooks/aws-parallelcluster-platform/resources/arm_pl/partial/_arm_pl_common.rb

```diff
@@ -162,3 +171,9 @@
 
   node_attributes "dump node attributes"
 end
+
+action_class do
+  def modulefile_dir
+    '/usr/share/Modules/modulefiles'
```

Comment:
Shouldn't `Modules` be `modules`?

Reply:
Only for ubuntu. See https://github.com/search?q=repo%3Aaws%2Faws-parallelcluster-cookbook%20%2Fusr%2Fshare%2FModules%2Fmodulefiles&type=code

Reply:
Alright, thanks for the info!